.github/workflows/iroha2-dev-pr.yml

```diff
@@ -66,7 +69,10 @@ jobs:
           name: lcov.info
           path: lcov.info
 
-  integration:
+  # migrating to pytest
```

Comment:
#4500 we're not migrating to pytest, rather pytest is an addition to iroha integration tests. The reason is that it's too involving for Rust devs to write pytests with allure. We would also want to move pytests out of this repository